translation/italian.json5

```diff
@@ -260,52 +260,52 @@
+        "NAN_INF_Infinity": "Infinity non è previsto dalle specifiche JSON originali",
+        "NAN_INF_NaN": "NaN non non è previsto delle specifiche JSON originali",
         // ============== JSONC messages ==============
-        "JSONC_JAVASCRIPT_COMMENT": "I commenti JavaScript non fanno parte della specifica JSON originale",
+        "JSONC_JAVASCRIPT_COMMENT": "I commenti JavaScript non sono previste dalle specifica JSON originali",
         // ============== JSON5 messages ==============
         "JSON5_WHITESPACE_CHAR": "I caratteri di spazio diverso da ' ', '\\t', '\\r' e '\\n' sono consentiti solo in JSON5",
-        "JSON5_SINGLEQUOTED_STRING": "Le stringhe tra virgolette singole sono consentite solo in JSON5",
-        "JSON5_ESCAPED_NEWLINE": "I caratteri di nuova riga con escape sono consentiti solo in JSON5",
-        "JSON5_X_ESCAPE": "\\x escape sono consentiti solo in JSON5",
-        "JSON5_ESCAPED_CHAR": "Solo in JSON5 è possibile eseguire l'escape del carattere '{0}'.",
-        "JSON5_UNQUOTED_KEY": "Le chiavi senza virgolette sono supportate solo in JSON5",
-        "JSON5_NUM_LEADING_PLUS": "I segni + iniziali nei numeri non sono consentiti tranne che in JSON5",
-        "JSON5_HEX_NUM": "I numeri esadecimali sono solo una parte di JSON5",
-        "JSON5_NUM_LEADING_DECIMAL_POINT": "I numeri con un punto decimale iniziale sono solo una parte di JSON5",
+        "JSON5_SINGLEQUOTED_STRING": "Le stringhe tra apici singoli sono consentite solo in JSON5",
+        "JSON5_ESCAPED_NEWLINE": "I caratteri di escape per &quot;a capo&quot; sono consentiti solo in JSON5",
+        "JSON5_X_ESCAPE": "I codici escape \\x sono consentiti solo in JSON5",
+        "JSON5_ESCAPED_CHAR": "Il codice Escape '{0}' è consentito solo in JSON5.",
+        "JSON5_UNQUOTED_KEY": "Le chiavi senza virgolette sono consentite solo in JSON5",
+        "JSON5_NUM_LEADING_PLUS": "I numeri preceduti dai simboli + sono consentiti solo in JSON5",
+        "JSON5_HEX_NUM": "I numeri esadecimali sono consentiti solo in JSON5",
+        "JSON5_NUM_LEADING_DECIMAL_POINT": "I numeri con un punto decimale iniziale sono consentiti solo in JSON5",
         "JSON5_COMMA_AFTER_LAST_ELEMENT_ARRAY": "Virgola dopo l'ultimo elemento dell'array",
         "JSON5_COMMA_AFTER_LAST_ELEMENT_OBJECT": "Virgola dopo l'ultima coppia chiave-valore dell'oggetto",
-        "JSON5_NUM_TRAILING_DECIMAL_POINT": "I numeri con un punto decimale finale sono solo una parte di JSON5",
+        "JSON5_NUM_TRAILING_DECIMAL_POINT": "I numeri con un punto decimale finale sono consenti solo in JSON5",
         // ============== BAD messages ==============
         "BAD_UNTERMINATED_MULTILINE_COMMENT": "Commento su più righe senza terminazione",
-        "BAD_PYTHON_COMMENT": "I commenti '#' in stile Python non fanno parte di alcuna specifica JSON ben accettata",
-        "BAD_STRING_CONTAINS_NEWLINE": "La stringa letterale contiene una nuova riga",
```

Comment:
I assume `"a capo"` is the way to say `newline` in Italian?

In any case, make sure all instances of `&quot;` are converted to `\"`.

Reply:
@molsonkiko  in my opinion "a capo"  is the best translation. 
It is the short form of  "ritorno a capo" that is similar to "carriage return" (e.g. #13#10) 
https://context.reverso.net/traduzione/italiano-inglese/ritorno+a+capo
It is used for "carriage return", newline, wordwrap
"nuova riga" is the translation of "new line", but not for "newline"